### ISSUE TITLE
display the webgme-taxonomy version information

### ISSUE THREAD

**Issue by yogeshVU** [webgme/webgme-taxonomy#451]
It will be good to display version information of the software used, similar to the webgme version information.

**Comment by brollb** [comment]
How about an endpoint to display this information since it will likely be used primarily by taxonomy designers/maintainers and will be easier than forking the footer widget to customize the version text. (It would be nice if there were a way to do this in a generic way for any webgme app.)

**Comment by yogeshVU** [comment]
yes, endpoint will be good to provide this information. Perhaps we could also add this information as a part of the dashboard.

**Comment by yogeshVU** [comment]
perhaps also related to this: https://github.com/webgme/webgme-taxonomy/issues/446#issue-1950233882
